build/ci/phase-template.yml

```diff
@@ -6,7 +6,8 @@ parameters:
   customMatrixes: ''
 
 phases:
-  - phase: ${{ parameters.name }}
+- ${{ if ne(variables['codeCoverageBuild'], 'true') }}:
+    - phase: ${{ parameters.name }}
     variables:
```

Comment:
You would need to indent everything under these if blocks in this file if you go with the current structure of the phase blocks. 

That is why the build is failing: 

```
phase-template.yml (Line: 11, Col: 5, Idx: 199) - (Line: 11, Col: 5, Idx: 199): While parsing a block collection, did not find expected '-' indicator.
```